nbval/plugin.py

```diff
@@ -437,12 +437,18 @@ def compare_outputs(self, test, ref, skip_compare=None):
             )
             return False
         elif test_keys - ref_keys:
+            extra_keys = test_keys - ref_keys
             self.comparison_traceback.append(
                 cc.FAIL
-                + "Unexpected output fields from running code: %s"
-                % (test_keys - ref_keys)
+                + "Unexpected output fields from running code: %s" % extra_keys
                 + cc.ENDC
-            )
+                + "<<<<<<<<<<<< Unexpected content:"
+                + "\n".join("%s: %s" % (i, testing_outs[i]) for i in extra_keys)
+                + cc.FAIL
+                + '>>>>>>>>>>>>'
+                + cc.ENDC
+                )
+            print('IN STDERR WE GOT: %s' % testing_outs['stderr'])
```

Comment:
Was this ALLCAPS print statement meant to be kept in?

Reply:
uoops no! that's clearly a leftover. Pushing a fix+rebase shortly